game/theater/conflicttheater.py

```diff
@@ -356,9 +364,14 @@ def add_preset_locations(self) -> None:
 
         for group in self.ewrs:
             closest, distance = self.objective_info(group)
-            closest.preset_locations.ewrs.append(
-                PointWithHeading.from_point(group.position, group.units[0].heading)
-            )
+            if distance < self.BASE_DEFENSE_RADIUS:
+                closest.preset_locations.ewrs.append(
+                    PointWithHeading.from_point(group.position, group.units[0].heading)
+                )
+            else:
+                closest.preset_locations.strike_locations.append(
```

Comment:
I noticed that my blue (non required) ewr locations were turning into generic strike locations, is this the culprit?  Should this say `closest.preset_locations.ewrs.append(` instead?

Reply:
Just tried it, answer is yes.